components/automate-ui/src/app/entities/desktop/desktop.reducer.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { desktopEntityReducer, DesktopEntityState, desktopEntityInitialState } from './desktop.reducer';
+import { UpdateDesktopsFilterPageSizeAndCurrentPage } from './desktop.actions';
+
+describe('desktopEntityReducer', () => {
+  const initialState: DesktopEntityState = desktopEntityInitialState;
+
+  describe('DesktopActionTypes', () => {
+    describe('UPDATE_DESKTOPS_FILTER_PAGE_SIZE_AND_CURRENT_PAGE', () => {
+
+      it('updates the pageSize and pageNumber', () => {
```

Comment:
This test is a little wonky - It works but i'm not sure it truly shows that state is being updated

I feel it looks funky because state updates return a new item so i'm not totally sure how to show that the state was different previously